layers/meta-resin-raspberrypi/recipes-kernel/linux/linux-raspberrypi/0003-media-uvcvideo-Fix-driver-reference-counting.patch

```diff
@@ -1,68 +0,0 @@
-From f9ffcb0a21e1fa8e64d09ed613d884e054ae8191 Mon Sep 17 00:00:00 2001
```

Comment:
We need some git log info on why this was removed

Reply:
This is the commit that did not remove this file as necessary: https://github.com/balena-os/balena-raspberrypi/commit/9b5833770c6cba9ac63cdbe76c3ee1fd3fbb409b#diff-13b710abbcab4410708b1190381ce1ef

Any suggestions for improvement of my commit message?

Reply:
I see; guess you can just say it's now in upstream and that should be fine.
Thanks

Reply:
Done.